helm/druid/Chart.yaml

```diff
@@ -13,10 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: v1
+apiVersion: v2
 appVersion: 0.19.0
 description: DEPRECATED Apache Druid is a high performance real-time analytics database.
 name: druid
+dependencies:
+  - name: zookeeper
+    version: 2.1.4
+    repository: https://charts.helm.sh/incubator
+    condition: zookeeper.enabled
+  - name: mysql
+    version: 1.6.4
+    repository: https://charts.helm.sh/stable
+    condition: mysql.enabled
+  - name: postgresql
+    version: 8.6.4
+    repository: https://charts.helm.sh/stable
+    condition: postgresql.enabled
 version: 0.2.18
```

Comment:
I think we need to bump version

Reply:
oh, we need to bump version. BTW, since migrating from v2 to v3, I think we can bump to `0.3.0`